ccxt/ccxt
# binance ws - reset connection

Binance requires to reset the connection every 24hrs.
This PR proposes to reject expected futures and throws a ResetConnection error so the user can catch it, and recall the watch function to create a reconnect.

It also updates the cli examples to print the `ResetConnection` and continue.

Easiest way to test:
1. set `'resetConnectionInterval': 3000`
4. See it close the connection after 3 seconds and reconnect again
